docker/conf/docker-sync.yml.dist

```diff
@@ -3,25 +3,25 @@ syncs:
     shopsys-framework-sync:
         sync_userid: 501
         src: './'
-        sync_excludes: [
-            '.ci',
-            '.git',
-            '.github',
-            '.idea',
-            '.docker-sync',
-            '.DS_Store',
-            'docker',
-            'nbproject',
-            'project-base/docker',
-            'project-base/docs',
-            'project-base/kubernetes',
-            'project-base/node_modules',
-            'project-base/var/cache',
-            'project-base/var/elasticsearch-data',
-            'project-base/var/postgres-data',
-            'project-base/web',
-            'vendor'
-        ]
+        sync_excludes:
+            - .ci
+            - .git
+            - .github
+            - .idea
+            - .docker-sync
+            - .DS_Store
+            - docker
+            - microservices
```

Comment:
there are no microservices anymore

Reply:
well, this should be in a separate PR :)

Reply:
:+1: fixed